src/osuTK.iOS/IOSAppDelegate.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using UIKit;
+using Foundation;
+using osuTK.Input;
+using osuTK.Platform;
+
+namespace osuTK.iOS
+{
+    public abstract class IOSAppDelegate : UIApplicationDelegate
+    {
+       
+        public abstract override bool FinishedLaunching(UIApplication application, NSDictionary launchOptions); // Exposing FinishedLaunching for osu-framework to handle
+
+        private readonly FileDropEventArgs FileDropArgs = new FileDropEventArgs();
+
+
+
+        public override bool OpenUrl(UIApplication app, NSUrl url, NSDictionary options) // File opening from external source. (Required Info.plist in game project)
+        {
+            var e = FileDropArgs;
```

Comment:
this is just wrong. why aren't you making a new instance of `FileDropEventArgs`?

Reply:
That was indeed incorrect. When I made this commit did not know how the eventhandler worked.